modules/xmpp/SDPUtil.js

```diff
@@ -443,6 +443,30 @@ var SDPUtil = {
             .split(" ")
             .map(ssrcStr => parseInt(ssrcStr));
     },
+
+    /**
+     * Sets the given codecName as the preferred codec by
+     *  moving it to the beginning of the payload types
+     *  list (modifies the given mline in place)
+     * @param {object} videoMLine the video mline object from
+     *  an sdp as parsed by transform.parse
+     * @param {string} the name of the preferred codec
+     */
+    preferVideoCodec: function(videoMLine, codecName) {
+        let payloadType = null;
```

Comment:
this is a good point...but what do you think about keeping it simple for now and modifying it once we hit scenarios where there are more?  i don't want to make it too complicated now